homeassistant/components/utility_meter/sensor.py

```diff
@@ -189,8 +192,7 @@ def async_source_tracking(event):
                 async_track_state_change(self.hass, self._tariff_entity,
                                          self.async_tariff_change)
 
-                component = self.hass.data[DATA_UTILITY][UTILITY_COMPONENT]
-                tariff_entity = component.get_entity(self._tariff_entity)
+                tariff_entity = self.hass.states.get(self._tariff_entity)
```

Comment:
This is a state we're getting, so the local variable name should reflect that. Calling it entity is misleading.